packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -448,6 +453,17 @@ contract SystemConfig is ProxyAdminOwnedBase, OwnableUpgradeable, Reinitializabl
         emit ConfigUpdate(VERSION, UpdateType.OPERATOR_FEE_PARAMS, data);
     }
 
+    function setDAFootprintGasScalar(uint16 _daFootprintGasScalar) external onlyOwner {
+        _setDAFootprintGasScalar(_daFootprintGasScalar);
+    }
```

Comment:
Let's add tests for this method to `SystemConfig.t.sol` using the same convention we do for the min `setMinBaseFee` tests